protocol/avdl/rekey.avdl

```diff
@@ -32,6 +38,12 @@ protocol rekey {
     array<ProblemTLF> tlfs;
   }
 
+  record ProblemSetDevices {
+    User user;
+    KID kid;
+    array<ProblemTLFDevices> tlfs;
```

Comment:
I was trying to make it so electron could just display the data it needs to display like the interface mockup in zeplin and not need to worry about what a KID is and how it relates to a Device.

@chrisnojima do you have any preference?